source/dem/dem.cc

```diff
@@ -594,6 +648,21 @@ DEMSolver<dim>::finish_simulation()
     }
 }
 
+template <int dim>
+void
+DEMSolver<dim>::update_floating_mesh_info()
+{
+  for (unsigned int i_solid = 0; i_solid < n_solids; ++i_solid)
+    {
+      floating_mesh_translational_velocity.insert(
+        {i_solid, solids[i_solid]->get_translational_velocity()});
```

Comment:
Why do you need to store these velocities in a seperate container? Why not just get them directly from the object when you are going to be doing the force calculation?